Tools/PlatformWPE.cmake

```diff
@@ -51,6 +51,12 @@ if (ENABLE_COG)
         set(COG_MESON_BUILDTYPE debugoptimized)
     endif ()
 
+    if (ENABLE_SANITIZERS)
+        set(COG_MESON_ASAN_OPTION "${ENABLE_SANITIZERS}")
```

Comment:
Is this correct, though? It looks like you enable asan in cog if _any_ sanitizer is enabled in WebKit, like, say, tsan?

Reply:
No we just pass the string through. `-DENABLE_SANITIZERS=address,thread` -> `-Db_sanitize=address,thread`. They support the same values as they are just passed to clang/gcc the same way.

Reply:
I forgot to rename the variable to something like `COG_MESON_SANITIZER_OPTION`.